lab3/ans/config/smfcfg.yaml

```diff
@@ -0,0 +1,16 @@
+configuration:
+  pfcp: # the IP address of N4 interface on this SMF (PFCP)
+    # addr config is deprecated in smf config v1.0.3, please use the following config
+    nodeID: smf.n4.org # the Node ID of this SMF
+    listenAddr: smf.n4.org # the IP/FQDN of N4 interface on this SMF (PFCP)
+    externalAddr: smf.n4.org # the IP/FQDN of N4 interface on this SMF (PFCP)
+  userplaneInformation: # list of userplane information
+    upNodes: # information of userplane node (AN or UPF)
+      UPF: # the name of the node
+        type: UPF # the type of the node (AN or UPF)
+        nodeID: upf.n4.org # the Node ID of this UPF
+        addr: upf.n4.org # the IP/FQDN of N4 interface on this UPF (PFCP)
+        interfaces: # Interface list for this UPF
+          - interfaceType: N3 # the type of the interface (N3 or N9)
+            endpoints: # the IP address of this N3/N9 interface on this UPF
+              - 10.100.3.100
```

Comment:
Can we write `upf.n3.org`?

Reply:
SMF is not in N3 net. It cannot resolve this alias. So, I use static ip address.